src/main/java/http/router/Route.java

```diff
@@ -0,0 +1,23 @@
+package http.router;
+
+import http.helper.Handler;
+
+import java.util.*;
+
+public class Route {
```

Comment:
this is very good 👏 Simple, optimized via hashmaps, easy to understand.

Reply:
Thank you 👍